apps/portals/src/types/portal-util-types.ts

```diff
@@ -44,6 +44,7 @@ export type DetailsPageContent =
 
 export type DetailsPageProps = DetailsPageContent & {
   showMenu?: boolean // default to true
+  /** Note, to exclude from processing, use the suffix "NotAQueryFilterKey" */
```

Comment:
I wonder if we could use [Template Literal Types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) to enforce a prefix or suffix